goosebit/settings/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .const import BASE_DIR, DB_MIGRATIONS_LOC, PWD_CXT, SECRET  # noqa: F401
+from .schema import GooseBitSettings
+
+config = GooseBitSettings()
+print(config)
```

Comment:
These print statements should be removed, no?

Reply:
Ha yeah, might need a pre-commit linter for `get rid of random print statements`...